src/interactive.sh

```diff
@@ -0,0 +1,40 @@
+function create_ssh_key()
```

Comment:
I think this code should be inside `init.sh`, I don't see why we want a new file for handling this.

Document this function.

Reply:
Should we move all the code in interactive.sh to init.sh? We thought of interactive as a sort of auxiliary file to init.

Reply:
Yes, move all the `interactive.sh` code inside `init.sh`.